src/components/ScrollableForm.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { DependencyList, FC, ReactNode, useEffect, useRef } from "react";
+import useEventListener from "@use-it/event-listener";
+import { getAbsoluteHeightBelow, getParentsBottomSpacing } from "util/helpers";
+
+interface Props {
+  children: ReactNode;
+  dependencies: DependencyList;
+  belowId?: string;
+}
+
+const ScrollableForm: FC<Props> = ({
+  dependencies,
+  children,
+  belowId = "",
+}) => {
+  const ref = useRef<HTMLDivElement>(null);
+
+  const updateFormHeight = () => {
+    const form = ref.current?.children[0];
+    if (!form) {
+      return;
+    }
+    const above = form.getBoundingClientRect().top + 1;
+    const below = getAbsoluteHeightBelow(belowId);
+    const parentsBottomSpacing = getParentsBottomSpacing(form as HTMLElement);
+    const offset = Math.ceil(above + below + parentsBottomSpacing);
+    const style = `height: calc(100vh - ${offset}px); min-height: calc(100vh - ${offset}px)`;
+    form.setAttribute("style", style);
+  };
+
+  useEventListener("resize", updateFormHeight);
+  useEffect(updateFormHeight, [...dependencies, ref]);
+
+  return (
+    <div ref={ref} className="scrollable-form">
+      <div className="details">{children}</div>
```

Comment:
Sorry for the nitpick after asking to pull the details div into this component. The `details` is quite generic, better `form-details`? Or if we go with the generic `ScrollableComponent`, maybe find better suiting class names.

Reply:
yeah, I renamed the class to `content-details`